src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_params.py

```diff
@@ -231,6 +234,29 @@ def register_source_uri_arguments(scope):
 for item in ['check-name', 'delete', 'list', 'show', 'show-usage', 'update', 'keys']:
     register_cli_argument('storage account {}'.format(item), 'account_name', account_name_type, options_list=('--name', '-n'))
 
+
+def replace_the_first(target, replacement):
+    def _wrap(arg):
+        arg.type.settings['help'] = arg.type.settings['help'].replace(target, replacement, 1)
```

Comment:
I prefer the current approach guarantees the help doc to be consistent with the SDK's docstring. If help is overridden in `help.py` as a result every time the SDK dependencies are upgraded all the help will need manual validate to ensure if they're still up-to-date. Given the change here is to leave the important information of options intact, I don't think override help is proper.